public/index.html

```diff
@@ -52,7 +52,7 @@
 			property="og:image"
 			content="https://devprotocol.xyz/assets/img/og.png"
 		/>
-		<meta property="og:description" content="It's Time to Tokenize OSS." />
+		<meta property="og:description" content="It's Time to Tokenize Open source software." />
```

Comment:
Looks good to me, but can you uppercase the s?

Ex.:
- **Open Source Software**

Reply:
Yes :)

Reply:
How about this line

Reply:
It would be enhance organic traffic  on your website with the help SEO .